exercises/es/test_general.py

```diff
@@ -28,19 +28,20 @@ def test_01_09_predictions(nlp):
     text = "Los Olímpicos de Tokio 2020 son la inspiración para la nueva colección de zapatillas adidas ZX."
     doc = nlp(text)
     ents = [(ent.text, ent.label_) for ent in doc.ents]
-    assert len(ents) == 1
-    assert ents[0] == ("Olímpicos de Tokio 2020", "MISC")
+    assert len(ents) == 2
+    assert ents[0] == ("Olímpicos de Tokio 2020", "LOC")
+    assert ents[1] == ("ZX", "MISC")
     assert doc[14].ent_type == 0
-    assert doc[15].ent_type == 0
+    assert doc[15].ent_type == 7654241940133152407
```

Comment:
The `ent_type === 0` check here is mostly a simple way to check that "this token has no entity type and is not part of any entity". So the last line here is kinda redundant, because that's already covered by asserting that there's an entity "ZX" with the label `MISC`.

Btw, don't the predictions here conflict with the actual exercise? The objective in 1.9 is that there's a missing entity that needs to be created, but now the model already predicts the partial span `ZX` as an entity. I think we need to find an example where all tokens in the span are not part of any entitiy.

(Otherwise, the concept of "btw, you can actually just add a span to the `doc.ents` manually" wouldn't work, because spaCy wouldn't allow you to add a new entity if an overlapping entity already exists.)